src/bin/vip-import-validate-files.js

```diff
@@ -64,95 +95,66 @@ command( { requiredArgs: 1, format: true } )
 		}
 
 		/**
-		 * Media file extension validation
-		 *
-		 * Ensure that prohibited media file types are not used
+		 * Get Media Import configuration
 		 */
+		const api = API();
```

Comment:
If all we're doing is passing the `api` object to the `mediaImportGetConfig` function, can we just initialize an `API` instance in that function itself? We may want to avoid passing this as a function?

Reply:
Yes, it would be better.

Reply:
Resolved in d1d7a9721a0b02669569140b08a75b9fac8eec3e